users/stanrc85/rules.mk

```diff
@@ -0,0 +1,12 @@
+TAP_DANCE_ENABLE = yes
+EXTRAKEY_ENABLE = yes
+RGBLIGHT_ENABLE = yes
+BACKLIGHT_ENABLE = no
+COMMAND_ENABLE = no
+BOOTMAGIC_ENABLE = no
+MOUSEKEY_ENABLE = no
+AUDIO_ENABLE = no
+CONSOLE_ENABLE = no
+NKRO_ENABLE = no
+
+SRC += stanrc85.c
```

Comment:
If you're not doing anything in the C file, you could remove this and the actual file, actually.  

I think.

Reply:
I was using it kind of a placeholder for now, and I didn't want to risk breaking anything if I took it out during my initial setup.

Reply:
It doesn't check for a C file.  It actually looks for the path, and the rules.mk.  And adds the userspace folder to the "path", so you don't need to add the full path name. 

So, there is no issue with removing this line, nor the c file from the folder.  

But if you want to leave it, that's absolutely fine.

Reply:
Good to know! In that case I'll clean it up and remove them.

Reply:
In case your curious, here is the detection/addition code:
https://github.com/qmk/qmk_firmware/blob/master/build_keyboard.mk#L288-L297

And the VPATH additino:
https://github.com/qmk/qmk_firmware/blob/master/build_keyboard.mk#L322